pkg/apis/greenhouse/v1alpha1/pluginconfig_types.go

```diff
@@ -137,6 +141,15 @@ type PluginConfig struct {
 	Status PluginConfigStatus `json:"status,omitempty"`
 }
 
+// GetReleaseNamespace returns the release namespace of the pluginconfig.
+func (p *PluginConfig) GetReleaseNamespace() string {
+	// TODO: Remove this method once all PluginConfigs have the ReleaseNamespace set.
```

Comment:
Also in consideration of the suggestion of adding a default namespace to the _PluginDefinitions_
```suggestion
	// TODO: Replace this method with a Defaulter in the webhook once all PluginConfigs have the ReleaseNamespace set.
```